app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -1,487 +1,627 @@
+        android:layout_marginRight="@dimen/multimeter_square_center_vertical_margin_right"
+        android:layout_marginStart="@dimen/multimeter_square_center_vertical_margin_left"
+        android:background="@android:color/black"
+        app:layout_constraintBottom_toBottomOf="@+id/knob_line24"
+        app:layout_constraintEnd_toStartOf="@+id/view2"
+        app:layout_constraintStart_toEndOf="@+id/knob_line"
+        app:layout_constraintTop_toTopOf="@+id/knob_line24" />
+
+    <View
+        android:id="@+id/knob_line6"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/multimeter_knoblines_width"
+        android:layout_alignParentBottom="true"
+        android:layout_marginBottom="@dimen/multimeter_square_horizontal_line_margin_bottom"
+        android:layout_marginEnd="@dimen/multimeter_square_margin_right"
+        android:layout_marginLeft="@dimen/multimeter_square_margin_left"
+        android:layout_marginRight="@dimen/multimeter_square_margin_right"
+        android:layout_marginStart="@dimen/multimeter_square_margin_left"
+        android:background="@android:color/black"
+        app:layout_constraintBottom_toTopOf="@+id/reset"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <View
+        android:id="@+id/view2"
+        android:layout_width="@dimen/multimeter_knoblines_width"
+        android:layout_height="@dimen/multimeter_square_vertical_line_height"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_marginEnd="@dimen/multimeter_square_margin_right"
+        android:layout_marginRight="@dimen/multimeter_square_margin_right"
+        android:background="@android:color/black"
+        app:layout_constraintBottom_toTopOf="@+id/knob_line6"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/kno_line" />
+
+    <it.beppi.knoblibrary.Knob
+        android:id="@+id/knobs"
+        android:layout_width="@dimen/multimeter_knob_width"
+        android:layout_height="@dimen/multimeter_knob_height"
+        android:layout_centerHorizontal="true"
+        android:layout_marginEnd="@dimen/multimeter_knob_margin_right"
+        android:layout_marginLeft="@dimen/multimeter_knob_margin_left"
+        android:layout_marginRight="@dimen/multimeter_knob_margin_right"
+        android:layout_marginStart="@dimen/multimeter_knob_margin_left"
+        android:layout_marginTop="@dimen/multimeter_knob_margin_top"
+        android:layout_weight="1"
+        android:rotation="90"
+        app:kKnobCenterColor="@color/colorPrimaryDark"
+        app:kKnobColor="@color/white"
+        app:kKnobRelativeRadius="0.99"
+        app:kNumberOfStates="14"
+        app:kSelectedStateMarkerColor="@color/white"
+        app:kStateMarkersColor="#00000000"
+        app:kStateMarkersWidth="1dp"
```

Comment:
Can't these color and dimen go in resource as well?

Reply:
Sure